examples/gno.land/p/sys/vals/types/types.gno

```diff
@@ -0,0 +1,50 @@
+package types
+
+import (
+	"std"
+)
+
+// Protocol defines the validator set protocol (PoA / PoS / PoC / ?)
+type Protocol interface {
```

Comment:
```suggestion
type Valset interface {
```

up to you.

Reply:
I agree the `Protocol` naming is super vague

I've swapped it to `ValsetProtocol`:
[53de738](https://github.com/gnolang/gno/pull/2130/commits/53de7381a100a1c1220fdc7e17aea69d2e95c9fa)